tests/Functional/tests/list-alternate-syntax/list-alternate-syntax.rst

```diff
@@ -2,10 +2,17 @@ Alternate List Syntax
 ---------------------
 
 -
-    Alternate list item 1
-    multiline
+    Lists may start on the next line after
 -
-    Alternate list item 2
+    In all cases, the first text determines
+    the level of indentation
 -
-    Alternate list item 3
-    multiline
+  As such, this is a paragraph.
+
+    Yet, this is considered to be blockquote.
+
+  And this is normal text again
+-
+    But with a more indented first line,
+    this same level is a normal paragraph
+  and this is not part of the list (warning)
```

Comment:
Sphinx triggers a warning here. Can we trigger warnings in the parser for cases like this? (lists have to end with a blank line)

Reply:
If Sphinx does, I think we can.